templates flutter packages updates

kzawadi opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines in CONTRIBUTING.md

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**- What I did**
Bumped the templates @platform flutter packages
**- How I did it**
Referencing the newer pub.dev versions.
**- How to verify it**
Create a new Flutter app using at_app _cli_ and in pubspec.yaml the new packages will be referenced instead of the old ones
**- Description for the changelog**
Updated the templates Flutter packages to be generated by the _cli_
<!--
Write a short (one-line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->

kzawadi committed:
Update app.dart

Bumping the @platform packages to the newest. This will fix an issue with flutter_keychain (It's not used anymore)
there was a migration to the biometric_storage package... but the at_app cli was not referencing the newer versions of @platfrom packages
when creating a new app which leads to issues.

kzawadi committed:
Merge pull request #1 from kzawadi/kzawadi-patch-1

Update app.dart

XavierChanth commented:
That file is generated by the build pipeline, these changes will be reverted by the CI. The appropriate place to change these is in the templates folder under each template.yaml file.

xavierchanth reviewed (changes requested):
See my comment above

kzawadi commented:
Oh okay let me see

kzawadi committed:
templates flutter packages updates

- What I did
Bumped the templates @platform flutter packages
- How I did it
Referencing the newer pub.dev versions.
- How to verify it
Create a new Flutter app using at_app cli and in pubspec.yaml the new packages will be referenced instead of the old ones
- Description for the changelog
Updated the templates Flutter packages to be generated by the cli

kzawadi committed:
Update app.dart
